ed448/src/macros.rs

```diff
@@ -130,3 +130,60 @@ macro_rules! define_mul_assign_variants {
         }
     };
 }
+
+macro_rules! scalar_from_impls {
```

Comment:
@tarcieri I ended up duplicating that from primeorder. I don't know if it's worth the effort to move that to `elliptic-curve`

Reply:
That's fine